presto-product-tests-launcher/src/test/java/io/prestosql/tests/product/launcher/testcontainers/TestRyukImage.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.product.launcher.testcontainers;
+
+import io.prestosql.testing.AbstractTestDistributedQueries;
+import org.testcontainers.utility.TestcontainersConfiguration;
+import org.testng.annotations.Test;
+
+import java.lang.reflect.Constructor;
+import java.util.Properties;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.testng.Assert.assertTrue;
+
+public class TestRyukImage
+{
+    void fakeDependency()
```

Comment:
This is strange. Just add an exclusion for the dependency checker.

Reply:
Actually, it looks like this test could go in `presto-testing`?

Reply:
I wanted the exclusion to go away when we remove this test class.